### ISSUE TITLE
Visualizzare data pubblicazione degli articoli/circolari in home

### ISSUE THREAD

**Issue by Simsimus** [italia/design-scuole-wordpress-theme#714]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-scuole-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-scuole-wordpress-theme/blob/main/CONTRIBUTING.md)

### Proposta

Buongiorno, 
credo sia molto utile far comparire la data di pubblicazione degli articoli e circolari presenti in home.
Spesso si ricevono feedback in merito con la richiesta di tale dato che al momento è possibile visualizzare solo entrando nell'articolo/circolare oppure accedendo al menù Novità.
Credete sia possibile implementare tale visualizzazione?

### Motivazione e contesto

Migliorare la fruizione degli articoli/circolari

**Comment by sviluppoRobyone** [comment]
Mi chiedo, potrebbe essere un'opzione configurabile? @zetareticoli

**Comment by Simsimus** [comment]
Dal mio punto di vista, se diventasse opzionale, sarebbe ancora meglio così ogni scuola può decidere in base alle proprie esigenze, ma in linea di massima credo che la presenza della data, nel nuovo scenario home che prevede la presenza di più articoli, possa dare una visione di timing migliorativa.

**Comment by zetareticoli** [comment]
La data di pubblicazione dovrebbe essere sempre visibile, per dare modo a chi consulta gli articoli di capire quali sono i più recenti.

**Comment by sviluppoRobyone** [comment]
Concordo, è necessario allora cambiare la visualizzazione e trovare un posto per la data dal punto di vista di layout perché attualmente nei temi statici non è mai stata presa in considerazione

**Comment by Simsimus** [comment]
Una buona posizione credo che sia fra titolo e abstract o anche come prima cosa sopra il titolo.

**Comment by sviluppoRobyone** [comment]
Tra i dubbi:
- Finora non sono state previste icone, andiamo di testo e basta? (esempio nell'immagine)
- Prefisso data: "Pubblicato" [articolo] o "Pubblicata" [notizia]?
- Prefisso data: Teniamo "il" che potrebbe però avere qualche problema di lettura in caso ad esempio di "Pubblicato il 11 settembre" oppure mettiamo "in data" che però diventa forse troppo lungo?

![image](https://github.com/user-attachments/assets/5ea29fa8-00c5-4346-8852-7eb6a1c1a3cc)

**Comment by Simsimus** [comment]
Credo vada bene anche solo la data, perché è ugualmente comprensibile, compatta e si evitano problemi di lettura.
Sono indeciso se è meglio sopra o sotto il titolo (vedere esempio)...forse meglio sopra.

P.S. stessa cosa andrà fatta per le circolari

![data](https://github.com/user-attachments/assets/2cdc02cc-1bc1-4ac8-ae3b-42210d330b68)

**Comment by zetareticoli** [comment]
La data va posta dopo il titolo per ragioni di accessibilità e lettura da screen reader.

**Comment by Simsimus** [comment]
Ok, va benissimo anche sotto ;)

**Comment by sviluppoRobyone** [comment]
Sempre per questioni di chiarezza ci deve essere un'indicazione di che data parliamo, perché ce ne sono due, data di pubblicazione e data di aggiornamento presenti che girano per il sito...

**Comment by zetareticoli** [comment]
> Sempre per questioni di chiarezza ci deve essere un'indicazione di che data parliamo, perché ce ne sono due, data di pubblicazione e data di aggiornamento presenti che girano per il sito... o che mettiamo un'icona con il title ma non mi sembra essere in linea grafica col resto dove icone non ce ne sono

assolutamente sempre e solo data di pubblicazione, quella di aggiornamento può stare nella pagina foglia dedicata al contenuto.

**Comment by Cate55** [comment]
Buongiorno, vorrei proporre una soluzione per risolvere il problema della mancata visualizzazione delle date nelle card delle circolari in home.
Ho modificato il file /template-parts/single/card.circolare.php alla riga 10, aggiungendo il seguente codice per mostrare la data di pubblicazione delle circolari:
 ```
<?php`
global $post;
$numerazione_circolare = dsi_get_meta("numerazione_circolare", "", $post->ID);
$accesso_circolare = circolare_access($post->ID);
?>
<div class="card card-bg bg-white card-thumb-rounded">
    <div class="card-body">
        <div class="card-content">
        <!-- Inserimento data -->
                <div class="date">
                    <span class="day"><?php echo date_i18n("d", strtotime($post->post_date)); ?></span>
                    <span class="month"><?php echo date_i18n("M", strtotime($post->post_date)); ?></span>
                    <span class="year"><?php echo date_i18n("Y", strtotime($post->post_date)); ?></span>
                </div>
        <!-- Fine inserimento data -->
        <?php if($accesso_circolare != "false") { ?>
            <h3 class="h5"><a href="<?php echo get_permalink($post); ?>"><?php echo get_the_title($post); ?></a></h3>
            <small class="h6 text-greendark"><?php _e("Circolare ", "design_scuole_italia"); echo $numerazione_circolare; ?></small>
            <p><?php echo $post->_dsi_circolare_descrizione; ?></p>
        <?php } else { ?>
                <div class="card-article-content">
                <p class="font-weight-bold pl-2">Il contenuto della circolare numero <?php echo $numerazione_circolare ?> (<?php echo date_i18n("j F Y", strtotime($post->post_date)); ?>) è riservato.</p>
                </div>
        <?php } ?>
        </div>
    </div><!-- /card-body -->
</div><!-- /card -->
<?php
```
 
questo il risultato
![data-circolare](https://github.com/user-attachments/assets/9bea6a6a-44a5-4789-a3d2-6cf62352d855)

**Comment by Cate55** [comment]
> La data va posta dopo il titolo per ragioni di accessibilità e lettura da screen reader.

Scusami ho notato adesso la tua precisazione pertanto il codice va modificato per spostare il blocco data.
Ho aggiornato il codice spostando la visualizzazione della data sotto il titolo per migliorare l'accessibilità e la lettura con screen reader.

//Ecco il nuovo blocco di codice:
```<?php
global $post;
$numerazione_circolare = dsi_get_meta("numerazione_circolare", "", $post->ID);
$accesso_circolare = circolare_access($post->ID);
?>

<div class="card card-bg bg-white card-thumb-rounded">
    <div class="card-body">
        <div class="card-content">
            <?php if($accesso_circolare != "false") { ?>
                <h3 class="h5"><a href="<?php echo get_permalink($post); ?>"><?php echo get_the_title($post); ?></a></h3>
                <small class="h6 text-greendark"><?php _e("Circolare ", "design_scuole_italia"); echo $numerazione_circolare; ?></small>
                <p><?php echo $post->_dsi_circolare_descrizione; ?></p>
                <!-- Sposto qui il blocco della data sotto il titolo -->
                <!--Inserimento data-->
                <div class="date">
                    <span class="day"><?php echo date_i18n("d", strtotime($post->post_date)); ?></span>
                    <span class="month"><?php echo date_i18n("M", strtotime($post->post_date)); ?></span>
                    <span class="year"><?php echo date_i18n("Y", strtotime($post->post_date)); ?></span>
                </div>
                <!-- Fine inserimento data -->
            <?php } else { ?>
                <div class="card-article-content">
                    <p class="font-weight-bold pl-2">Il contenuto della circolare numero <?php echo $numerazione_circolare ?> (<?php echo date_i18n("j F Y", strtotime($post->post_date)); ?>) è riservato.</p>
                </div>
            <?php } ?>
        </div>
    </div><!-- /card-body -->
</div><!-- /card -->
<?php
```

**Comment by Simsimus** [comment]
Buongiorno,
la soluzione a tale skill entro quanto tempo pensate possa essere implementata?

**Comment by Simsimus** [comment]
Salve, 
mi sapete dire se questa modifica verrà applicata?
